framework/auth/core.py

```diff
@@ -147,10 +147,11 @@ def get_user(email=None, password=None, verification_key=None):
 class Auth(object):
 
     def __init__(self, user=None, api_node=None,
-                 private_key=None):
+                 private_key=None, private_link=None):
```

Comment:
Since presumably a `private_link` will have a `key` attribute (yes?), is it necessary to have both here? Can we get rid of `private_key` and just use `private_link.key` instead?